src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -642,6 +642,8 @@ public static bool TryPrintStackTraceFromDmp(string dmpFile, TextWriter outputWr
         // The children are sorted in the order they should be dumped
         static unsafe IEnumerable<Process> FindChildProcessesByName(Process process, string childName)
         {
+            Console.WriteLine($"Finding all child processes of '{process.ProcessName}' (ID: {process.Id}) with name '{childName}'");
```

Comment:
Everywhere else in this code I see an outputWriter being used but in a few spots I also see Console.WriteLine() being used. Can you tell what the distinction is? Its not clear to me when we want to use one and when we use the other.

Reply:
I believe the `outputWriter` + `errorWriter` are used to write each test's `output.txt` + `error.txt`.
I haven't yet pinpointed the logic for how each helix workitem's `console.<id>.log` is created, but it looks to me that the `console.<id>.log` will stitch together each individually generated xunit test wrapper's  `error.txt` and `output.txt`  in addition to the `Console.WriteLine`s used.

I think the `output.txt` would be used to validate the test result, `error.txt` would be used to capture any test specific failures, and any other failures that show up in `console.<id>.log` are probably non test-specific (i.e. infrastructure or timeout?) failures.

Reply:
> I believe the outputWriter + errorWriter are used to write each test's output.txt + error.txt.

Sounds reasonable to me. As long as the output will be visible in the CI output I'm happy but hopefully there is some owner for this code who can confirm we are following the intended convention :)

Reply:
I forced a timeout in the BinderTracingTests in a previous commit and the logging added in this PR did show up in the `console.*.log` file in CI.